tests/simulators/carla/test_carla.py

```diff
@@ -8,6 +8,18 @@
 )
 
 
+def test_map_param_parse(getAssetPath):
+    pytest.importorskip("carla")
```

Comment:
This test doesn't require the `carla` package, so let's remove this line. (Presumably this restriction is why we're getting the coverage warning below.)

Reply:
Ack! Will make that change now!